### ISSUE TITLE
Titles are truncated

### ISSUE THREAD

**Issue by equinusocio** [lelouch77/docusaurus-lunr-search#9]
I have a strange issue, I just installed this plugin but the search box shows truncated titles:

<img width="522" alt="Screenshot 2020-04-23 at 16 39 35" src="https://user-images.githubusercontent.com/10454741/80112106-063a9780-8581-11ea-900a-e449844674f7.png">

Properties and Usage become roperties# and sage#

**Comment by lelouch77** [comment]
Hi can you share the code that particular page ...

Thanks

**Comment by equinusocio** [comment]
This is the content, but I get the same error with all the page

```mdx
---
id: avatar
title: Avatar
description: ' '
---
import { DocHint } from '@site/src/components/doc-hint';
import { Avatar } from '@OvalMoney/components';
import { Layout } from '@site/src/components/layout';
import { PropsTable } from '@site/src/components/props-table';
import { LinkWidget } from '@site/src/components/link-widget';

The **Avatar** component is used to show the user's image in multiple sizes. The default size of the component is `40px`.

## Usage

```js
import { Avatar } from '@OvalMoney/components';
```

Once you import it, you can use the `<Avatar />` element inside your JSX template.

<Layout display="flex">
  <Avatar />
  <Avatar size={24} />
  <Avatar size={16}  />
  <Avatar image="https://bit.ly/2uxtDl3" />
  <Avatar size={24} image="https://bit.ly/2uxtDl3" />
  <Avatar size={16} image="https://bit.ly/2uxtDl3" />
</Layout>

```jsx
<Avatar />
<Avatar size={24} />
<Avatar size={16}  />
<Avatar image="https://bit.ly/2uxtDl3" />
<Avatar size={24} image="https://bit.ly/2uxtDl3" />
<Avatar size={16} image="https://bit.ly/2uxtDl3" />
```

## Properties

<DocHint type="directive">
  If you are using Typescript a **Avatar.d.ts** is already provided with the component.
</DocHint>

<PropsTable properties={[
  {
    name: 'className',
    type: 'string',
    default: '',
    required: false,
    description: 'Classes that will apply to the avatar element. It will be merged with the built-in classes.',
  },
  {
    name: 'size',
    type: '40 | 24 | 16',
    default: '40',
    required: false,
    description: 'The hint text. The content should be as short as possible.',
  },
  {
    name: 'image',
    type: 'string',
    default: '',
    required: false,
    description: 'The path of the image to display. If not present, it will shown a placeholder image',
  },
  {
    name: 'attributes',
    type: 'HTMLAttributes',
    default: '',
    required: false,
    description: 'You can pass all the [global html attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes#List_of_global_attributes).',
  },
]} />

## Related resources

<Layout direction="horizontal" repeat="fit" columnWidth="50%" align="start">
  <LinkWidget url="/docs/components/avatar" title="design" label="Avatar" />
</Layout>
```

**Comment by equinusocio** [comment]
@lelouch77 Let me know if you have room to fix it, otherwise we'll find another plugin 👍

**Comment by lelouch77** [comment]
Hi, Thanks for your response, try this version `npm i  docusaurus-lunr-search@1.0.3-rc1` and let me know is it fixed ... 

Thnaks

**Comment by equinusocio** [comment]
Yes, it seems to work now! Thank you!⛑

**Comment by lelouch77** [comment]
Cool thats great.. I will publish new version ... Thanks :)

**Comment by lelouch77** [comment]
fixed in version `1.0.3`
